app/services/analytics_events.rb

```diff
@@ -2817,6 +2817,13 @@ def telephony_otp_sent(
     )
   end
 
+  # Tracks When users visit the add phone page
+  def add_phone_setup_visit
+    track_event(
+      'Phone Setup visited',
```

Comment:
Minor: Capitalization consistency with other similar events.

```suggestion
      'Phone Setup Visited',
```